rocAL/rocAL_pybind/examples/rocAL_api_caffe_reader.py

```diff
@@ -28,104 +28,102 @@
-                                    ltrb=True,
-                                    allow_no_crop=True,
-                                    num_attempts=1)
-            images = fn.decoders.image_slice(jpegs, crop_begin, crop_size, output_type = types.RGB, path=image_path, annotations_file="", random_shuffle=True,shard_id=local_rank, num_shards=world_size)
+        if rocal_bbox:
+            jpegs, labels, bboxes = fn.readers.caffe(path=image_path, bbox=rocal_bbox)
+            images = fn.decoders.image(jpegs, path=image_path, shard_id=local_rank, random_shuffle=True)
 
         else:
-            jpegs, labels = fn.readers.caffe(path=image_path, bbox=_rocal_bbox, random_shuffle=True)
+            jpegs, labels = fn.readers.caffe(path=image_path, bbox=rocal_bbox)
             images = fn.decoders.image(jpegs, path=image_path, output_type=types.RGB, shard_id=local_rank, num_shards=world_size, random_shuffle=True)
 
-        images = fn.resize(images, resize_x=crop_size_resize,
-                           resize_y=crop_size_resize)
+        images = fn.resize(images, resize_width=224, resize_height=224, output_layout=tensor_layout)
         pipe.set_outputs(images)
     # Build the pipeline
     pipe.build()
     # Dataloader
-    data_loader = ROCALClassificationIterator(pipe , display=display, device=device, device_id=args.local_rank)
-
+    data_loader = ROCALClassificationIterator(pipe, display=0, device=device, device_id=args.local_rank)
     # Training loop
     cnt = 0
     # Enumerate over the Dataloader
     for epoch in range(args.num_epochs):  # loop over the dataset multiple times
         print("epoch:: ", epoch)
-        if not _rocal_bbox:
-            for i, (image_batch, labels) in enumerate(data_loader, 0):  # Classification
+        if not rocal_bbox:
+            for i, ([image_batch], labels) in enumerate(data_loader, 0):  # Classification
                 if args.print_tensor:
                     sys.stdout.write("\r Mini-batch " + str(i))
                     print("Images", image_batch)
                     print("Labels", labels)
                 for element in list(range(batch_size)):
                     cnt = cnt + 1
-                    draw_patches(image_batch[element], cnt)
+                    draw_patches(image_batch[element], cnt, args=args)
             data_loader.reset()
         else:
-            for i, (image_batch, bboxes, labels) in enumerate(data_loader, 0):  # Detection
-                if i ==0 :
+            for i, ([image_batch], bboxes, labels) in enumerate(data_loader, 0):  # Detection
+                if i == 0:
                     if args.print_tensor:
                         sys.stdout.write("\r Mini-batch " + str(i))
                         print("Images", image_batch)
                         print("Bboxes", bboxes)
                         print("Labels", labels)
                 for element in list(range(batch_size)):
                     cnt = cnt + 1
```

Comment:
cnt += 1 is better statement

Reply:
Changed all such instances to use += operator